### PR TITLE
Dfast updated

### PR DESCRIPTION
**Pull request by nigyta**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/dfast/meta.yaml [hunk 1]
```diff
@@ -3,29 +3,24 @@ package:
...
 source:
   url: https://github.com/nigyta/dfast_core/archive/1.2.3.tar.gz
   sha256: 710d2cad3874aba691e5995bc3865785e1807a09bf3a8a8c622126d7d4551859 
 
 requirements:
-  build:
-    - python >=3.5
+  host:
+    - python >=3.6
     - biopython
```
Comment (dpryan79) [inline recipes/dfast/meta.yaml]: You can actually remove the host section, nothing in here is used during installation.

File: recipes/dfast/meta.yaml [hunk 2]
```diff
@@ -3,29 +3,24 @@ package:
   version: 1.2.3
 
 build:
-  number: 0
-  noarch: generic
+  number: 1
+  noarch: python
```
Comment (dpryan79) [inline recipes/dfast/meta.yaml]: It doesn't really matter, but this is in fact `noarch: generic` (`noarch: python` is only needed for traditional python packages (i.e., things installed with `pip install`)).

Comment (dpryan79) [reply recipes/dfast/meta.yaml]: Actually, this should have never been a generic package, since it's vendoring `last` and probably some other packages :(

Comment (nigyta) [reply recipes/dfast/meta.yaml]: @dpryan79 Thank you for the comments.
Actually, I just wanted to add BLAST dependency ">=2.6.0", but build test fails for some reason despite that it succeeded last month.
I'm going to try again.

**Comment by dpryan79** [comment]
I expect the trick will be to add `last` to the run time dependencies and remove `bin/Linux` and `bin/Darwin`.

**Comment by nigyta** [comment]
@dpryan79
Well, the software package includes customized version of last, So, I cannot remove the `bin` folder.

**Comment by dpryan79** [comment]
@nigyta Ah, that's good to know, I had assumed it was just the stock `last`. If you can get this to work on Linux then just ping me and I'll get it working on OSX too. We'll have to remove the `noarch: generic` line and add the following to the end of the file since it's packaging some binaries:

    extra:
      skip-lints:
        - should_be_noarch_generic

**Comment by nigyta** [comment]
@dpryan79
Thank you! 
Yes, I want it works on both Linux and Mac. I just pushed the update. Hope it works.

**Comment by nigyta** [comment]
Sorry, I pushed again as I forgot to add openjdk dependency.

**Comment by dpryan79** [comment]
I think this will pass testing now (I've approved it if so, so it should be mergable).

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**